modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -4,7 +4,26 @@
 
 :_content-type: PROCEDURE
 [id="persistent-storage-csi-migration-enable_{context}"]
-= Enabling CSI automatic migration
+= Manually enabling CSI automatic migration
+
+If you want to test Container Storage Interface (CSI) migration in development or staging {product-title} clusters, in-tree to CSI migration must be manually enabled for the following in-tree volume types:
+
+* AWS Elastic Block Storage (EBS)
+
+* Google Compute Engine Persistent Disk (GCE-PD)
+
+* VMware vSphere Disk
+
+* Azure File
+
+CSI migration for these volume types is supported at the Technology Preview (TP) level.
```

Comment:
Usually stated this way. I am also wondering if perhaps you could just omit this line entirely since it's in the big red "READ ME" box?
```suggestion
CSI migration for these volume types is supported as a Technology Preview (TP) feature.
```

Reply:
Removed sentence as added this statement to TP boilerplate note.